applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -163,3 +149,118 @@ Test command
 ------------
 
 The test command is not supported.
+
+LwM2M Carrier library configuration #XCARRIERCFG
+================================================
+
+The ``#XCARRIERCFG`` command allows you to configure the LwM2M carrier library. The settings shall be stored and apply persistently.
+
+.. note::
+   To use ``#XCARRIERCFG``, the :kconfig:option:`CONFIG_LWM2M_CARRIER_SETTINGS` Kconfig option must be enabled.
+   For more details on library configuration, refer to :ref:`lwm2m_configuration`.
```

Comment:
```suggestion
   For more details on LwM2M carrier library configuration, see the :ref:`lwm2m_configuration` section of the library's documentation.
```
Just a little more detail since the link will be titled just "Configuration".